kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -143,7 +143,17 @@ public void open(Collection<TopicPartition> partitions) {
             config.getClass(PartitionerConfig.PARTITIONER_CLASS_CONFIG);
 
     Partitioner<FieldSchema> partitioner = partitionerClass.newInstance();
-    partitioner.configure(new HashMap<>(config.plainValues()));
+
+    HashMap<String, Object> plainValues = new HashMap<>(config.plainValues());
+    HashMap<String, ?> originals = new HashMap<>(config.originals());
```

Comment:
Okay no I remember why I did this. I was running into some weirdness with the Capture<?> values in the Hashmap. I'll see if I can find a more sensible workaround